DHFPROD-7872: Display results in table view

briantang opened the pull request:
### Description


#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added Release Notes

briantang commented:
@iveyMarklogic @aniamy Please verify that the previous issues you noticed are gone with this PR.

aniamy commented on marklogic-data-hub-central/ui/src/components/sidebar/sidebar.tsx:
```diff
@@ -34,6 +34,7 @@ interface Props {
   setHubArtifactsVisibilityPreferences: any;
   hideDataHubArtifacts: boolean;
   cardView: boolean;
+  graphView: boolean;
```
Are we using this boolean?
Can we use it in line 690? Show related with we have related and we are in graphView.
Also in line 91. Do not even process related if we are not in graph view.

briantang replied:
@aniamy Good find. I brought it in for that exact reason but planned to continue that effort as part of DHFPROD-8433 because I thought this PR was a higher priority. Since the PR hasn't gone through yet, I'll include the logic here.

briantang commented:
Added AC + tests for disabling related entity panel in table/snippet view, and also implemented a fix for issues from "Exploring Data" from the Run tile conflicting with the table changes.

xnikhil08 commented:
@briantang - Look like I am having trouble saving the query when i update the base entity and use searchText, like i showed in the screencast. If it's unrelated to your work here, then we can create a separate ticket for this issue.

xnikhil08 reviewed (approved):
Looks good. Except the issue with saving the query, as mentioned earlier, but it can be another ticket.

aniamy commented:
Grear work updating the tests but now there are many failing in the pipeline :(

briantang commented:
@aniamy Ah, I did additional clean up to remove the old "entity properties" panel now that we have the entity-specific sidebar and now it's causing some issues. Will update the tests.
